fabulaws-config.yml

```diff
@@ -78,11 +78,10 @@
     production:
     -
     staging:
-    - dualstack.opendebates-staging-lb-1737156337.us-east-1.elb.amazonaws.com
+    -
     testing:
-    - testing.demopenforum.com
-    - opendebates-testing.cakt.us
-    - dualstack.opendebates-testing-lb-1802486946.us-east-1.elb.amazonaws.com
+    - opendebates-testing-lb-476601241.us-east-1.elb.amazonaws.com
```

Comment:
Not an issue for this PR, but before we settle on a production ELB URL and commit it here, let's make sure security groups can be setup in VPC to allow traffic only from cloudflare (and our office IPs).